### PR TITLE
Cypress region id elements

### PR DESCRIPTION
**Pull request by SophieDePaula**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -78,7 +78,6 @@ jobs:
         if: ${{matrix.xvfb}}
         run: |
           sudo apt-get install -y xvfb
-          Xvfb -ac $DISPLAY -screen 0 1280x1024x16 &
```
Comment (kyryloonufriiev) [inline .github/workflows/test.yml:81]: What is wrong with that?

Comment (SophieDePaula) [reply .github/workflows/test.yml:81]: @kyryloonufriiev when I run the `test` workflow, I see this error message: ` [2571:0724/141402.139934:ERROR:ozone_platform_x11.cc(248)] Missing X server or $DISPLAY`. I thought removing this line will help, but it didn't so I'll undo this

File: js/packages/eyes-cypress/src/browser/eyesCheckMapping.js [hunk 2]
```diff
@@ -87,14 +87,14 @@ function eyesCheckMapValues({args, refer}) {
     if (!Array.isArray(regions)) regions = [regions];
     let resRegions = [];
     for (const region of regions) {
-      if (region.element) {
+      if (region.element || isHTMLElement(region) || region.jquery) {
```
Comment (kyryloonufriiev) [inline js/packages/eyes-cypress/src/browser/eyesCheckMapping.js:90]: what `region.jquery` means?

**Review by kyryloonufriiev — APPROVED** [review]

File: js/packages/eyes-cypress/src/browser/eyesCheckMapping.js [hunk 3]
```diff
@@ -93,6 +93,11 @@ function eyesCheckMapValues({args, refer}) {
           for (const refElement of currRefElements) {
             resRegions.push({region: refElement, padding: region.padding});
           }
+        } else if (region.regionId) {
+          let currRefElements = refElements(region.element);
+          for (const refElement of currRefElements) {
+            resRegions.push({region: refElement, regionId: region.regionId});
+          }
```
Comment (kyryloonufriiev) [inline js/packages/eyes-cypress/src/browser/eyesCheckMapping.js:100]: `regionId` and `padding` could be specified todether.

**Review by kyryloonufriiev — CHANGES_REQUESTED** [review]

**Comment by SophieDePaula** [comment]
@kyryloonufriiev I added some changes, so please have a look and if it's all good I'll merge it.

**Comment by kyryloonufriiev** [comment]
@SophieDePaula ☝🏼 Do you see the comment I left before?

**Comment by SophieDePaula** [comment]
> @SophieDePaula ☝🏼 Do you see the comment I left before?
@kyryloonufriiev I see my last commit, thanks.

File: js/packages/eyes-cypress/src/browser/eyesCheckMapping.js [hunk 4]
```diff
@@ -87,14 +87,21 @@ function eyesCheckMapValues({args, refer}) {
...
           let currRefElements = refElements(region.element);
           for (const refElement of currRefElements) {
-            resRegions.push({region: refElement, padding: region.padding});
+            let curr = {region: refElement};
+            if (region.padding) {
+              curr['padding'] = region.padding;
+            }
+            if (region.regionId) {
+              curr['regionId'] = region.regionId;
+            }
```
Comment (kyryloonufriiev) [inline js/packages/eyes-cypress/src/browser/eyesCheckMapping.js:100]:
```suggestion
            if (region.padding) {
              curr.padding = region.padding;
            }
            if (region.regionId) {
              curr.regionId = region.regionId;
            }
```

**Review by kyryloonufriiev — APPROVED** [review]

**Pull request merged**